apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/Trace/SimpleManagedTracer.cs

```diff
@@ -76,6 +76,9 @@ public void AnnotateSpan(Dictionary<string, string> labels)
             public ulong SpanId() => TraceSpan.SpanId;
         }
 
+        /// <summary>The number of nanoseconds in a millisecond.</summary>
+        internal static readonly int _nanosecondsInAMillisecond = 1000000; // 1,000,000
```

Comment:
Rewrite as 1_000_000 and then you don't need the comment :)

Reply:
I feel like you tell me this every 2-3 months and then I forget :-P